PactNet.Tests/PactUriOptionsTests.cs

```diff
@@ -6,31 +6,33 @@ namespace PactNet.Tests
     public class PactUriOptionsTests
     {
 
-        // This is an experimental test set.
-        // Use of Data Driving improves readability and 
-        // shows all the permutations run by one test
+        const string UsernameIsEmptyOrNullMessage = "username is null or empty.";
+        const string PasswordIsNullOrEmptyMessage = "password is null or empty.";
+        const string UsernameContainsColunMessage = "username contains a ':' character, which is not allowed.";
+
         [Theory]
-        [InlineData( ""        , ""            , "username is null or empty." ) ] 
-        [InlineData( ""        , "somepassword", "username is null or empty." ) ] 
-        [InlineData( "goodname", ""            , "password is null or empty." ) ] 
-        [InlineData( null      , ""            , "username is null or empty." ) ]
-        [InlineData( ""        , null          , "username is null or empty." ) ]
-        [InlineData( null      , null          , "username is null or empty." ) ]
-        [InlineData( "bad:name", ""            , "username contains a ':' character, which is not allowed." ) ] // RFC 2617 compliance
-        [InlineData( "bad:name", "goodpass"    , "username contains a ':' character, which is not allowed." ) ] // RFC 2617 compliance
-        [InlineData( "bad:name", null          , "username contains a ':' character, which is not allowed." ) ] // RFC 2617 compliance
-        [InlineData( "goodname", null          , "password is null or empty." ) ] // RFC 2617 compliance
-        [InlineData( "goodname", ""            , "password is null or empty." ) ] // RFC 2617 compliance
-        public void Ctor_takesInvalidUserNameandPasswordParamsThatThrowArgException(string username, string password, string expectedMessage)
-        {
-            Exception e = Assert.Throws <ArgumentException> (() => new PactUriOptions(username, password));
+        [InlineData( "bad:name", UsernameContainsColunMessage  ) ] // RFC 2617 compliance
+        [InlineData( ""        , UsernameIsEmptyOrNullMessage  ) ] 

```

Comment:
Something is not right with spacing here

Reply:
Hi Jacek,

I am not quite sure what you are thinking. I learnt to do that from a guy who mentored me in threaded software, to "tidy" the comma's so they align with the widest item. Happy to adjust the style.

Reply:
I don't know the .NET ecosystem very well, but in other languages there are standards that you can just adopt, set your IDE/editor to do on your behalf, and never think about. Something for us to consider on this project?

Reply:
@mefellows i have that switched on, this has to be managed by the coder. I'll remove the spaces :)